pkg/noderesourcetopology/least_numa.go

```diff
@@ -137,10 +137,10 @@ func nodesAvgDistance(numaNodes NUMANodeList, nodes ...int) float32 {
 
 	for _, node1 := range nodes {
 		for _, node2 := range nodes {
-			cost, ok := numaNodes[node1].Costs[node2]
+			cost, ok := numaNodes[node1].Costs[numaNodes[node2].NUMAID]
```

Comment:
(mostly doublechecking) can we ever go out of bounds here?

Reply:
Nope :)